README.md

```diff
@@ -60,6 +60,11 @@ as long as they're present in the channel mapping.
     "ircOptions": { // Optional node-irc options
       "floodProtection": false, // On by default
       "floodProtectionDelay": 1000 // 500 by default
+    },
+    "slackOptions": { // Optional node-irc options for Slack
```

Comment:
I think this doesn't have to be grouped here. The point of the grouping in `ircOptions` is just to pass it to the irc library.